backend/src/meals.js

```diff
@@ -99,6 +99,80 @@ async function existsMeal(client, week) {
     return output
 }
 
+// This function returns the correct list of clients 
+// that need deliveries for a given day
+router.post('/routeOverviewDay', (req, res) => {
+    // takes in these parameters from the front end.
+    // site is used to search and day which is a string M, T, W, Th, F
+    let {site, day} = req.body
+    
+    // this calls the Clients database and searches for matches to the site
+    // if there is an error it prints in the backend console 
+    // WARNING: danger of finding all clients with the same site
+    // if someone puts in "SLO" and another "San Luis Obispo" they will not be 
+    // found in the same list. This function will sort by SLO
+    Client.find({site: site}, function (err, clients) {
```

Comment:
Fetch data from meals database, not the client database. Route totals will change week by week.